### PR TITLE
Feat/async rework

### PR DESCRIPTION
**Pull request by weliasz**

### CODE REVIEW

File: splunk_connect_for_snmp_poller/manager/mib_server_client.py [hunk 1]
```diff
@@ -58,38 +56,24 @@ def get_translation(var_binds, mib_server_url, data_format):
...
-        )
-        adapter = HTTPAdapter(max_retries=retry_strategy)
-        session = requests.Session()
-        session.mount("https://", adapter)
-        session.mount("http://", adapter)
-        resp = session.post(
-            TRANSLATION_URL, headers=headers, data=payload, params=params, timeout=60
-        )
-
+        async with ClientSession() as session:
```
Comment (okuzhel-splunk) [inline splunk_connect_for_snmp_poller/manager/mib_server_client.py:64]: why do we need 
_async with ClientSession() as session_ if in trap we do the same with just session = ClientSession?

Comment (weliasz) [reply splunk_connect_for_snmp_poller/manager/mib_server_client.py:64]: It is Python equivalent of autoclosable feature. I missed it in trap. I'll add it now.

Comment (okuzhel-splunk) [reply splunk_connect_for_snmp_poller/manager/mib_server_client.py:64]: and same question as in trap, was removal of the retries a wanted change in logic?

**Review by okuzhel — APPROVED** [review]

**Review by omrozowicz-splunk — APPROVED** [review]

**Pull request merged**